src/utils/include/com/amazonaws/kinesis/video/utils/Include.h

```diff
@@ -1375,6 +1375,65 @@ PUBLIC_API SIZE_T getInstrumentedTotalAllocationSize();
 #define RESET_INSTRUMENTED_ALLOCATORS()             resetInstrumentedAllocatorsNoop()
 #endif
 
+//////////////////////////////////////////////////////////////////////////////////////////////////////
+// File logging functionality
+//////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/**
+ * File logger error values starting from 0x41300000
+ */
+#define STATUS_FILE_LOGGER_BASE                                             STATUS_UTILS_BASE + 0x01200000
+#define STATUS_FILE_LOGGER_INDEX_FILE_INVALID_SIZE                          STATUS_SEMAPHORE_BASE + 0x00000001
+
+/**
+ * File based logger limit constants
+ */
+#define MAX_FILE_LOGGER_STRING_BUFFER_SIZE          (3 * 1024 * 1024)
+#define MIN_FILE_LOGGER_STRING_BUFFER_SIZE          (10 * 1024)
+#define MAX_FILE_LOGGER_LOG_FILE_COUNT              (10 * 1024)
+
+/**
+ * Default values used in the file logger
+ */
+#define FILE_LOGGER_LOG_FILE_NAME                   "kvsFileLog"
+#define FILE_LOGGER_LAST_INDEX_FILE_NAME            "kvsFileLogIndex"
+#define FILE_LOGGER_STRING_BUFFER_SIZE              (100 * 1024)
+#define FILE_LOGGER_LOG_FILE_COUNT                  3
```

Comment:
Same with this? Perhaps make it a little more

Reply:
if ppl dont want the default setting they can always use createFileLogger.